models/todo.js

```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize(process.env.DATABASE_URL);
+
+const Todo = sequelize.define('todos', {
+  title: { type: Sequelize.STRING },
+  completed: { type: Sequelize.BOOLEAN },
+}, {
+  classMethods: {
+    archiveCompleted: async () => {
+      await Todo.destroy({ where: { completed: true } });
```

Comment:
No need to async and await here, just return Todo.destroy and you can async and await further up the chain